keycloak/themes/haven/login/login.ftl

```diff
@@ -54,6 +54,9 @@
             <#if realm.password && social.providers??>
                 <div id="kc-social-providers" class="mb-auto text-center">
                     <p class="mb-0 primary-font">Log In With:</p>
+                        <list>
+                            <p class="mb-1"><a href="/auth/realms/havendev/clients/magic-link/redirect" id="zocial-magic-link" class="zocial magic-link"> <span class="text">Get MagicLink</span></a></p>

```

Comment:
What do you think about changing this text to "Email me a sign-in link"?

Reply:
Good idea! Just changed it